Makefile

```diff
@@ -36,9 +36,9 @@ test-unit:
 
 .PHONY: test-connectivity
 test-connectivity:
-	SHARED_SERVICE_CLUSTER_KUBECONFIG=$(SHARED_SERVICE_CLUSTER_KUBECONFIG) \
-	WORKLOAD_CLUSTER_KUBECONFIG=$(WORKLOAD_CLUSTER_KUBECONFIG) \
-	ginkgo -v $(PWD)/test/connectivity
+	CLUSTER_ONE_KUBECONFIG=$(WORKLOAD_CLUSTER_KUBECONFIG) \
+	CLUSTER_TWO_KUBECONFIG=$(SHARED_SERVICE_CLUSTER_KUBECONFIG) \
```

Comment:
Nitpick - It seems to me the tests now deal with cluster_one and cluster_two - which is nice. But the e2e script and makefile still names the clusters as shared_services and workload. We _could_ make the e2e script and makefile consistent. I'm assuming that is ok, and there is no reason not to?

(I'm happy to do the work of renaming in the e2e and makefile BTW, and make that as a PR to this PR. Or commit on this PR. Or a separate PR entirely)

Reply:
I'm still a bit partial to giving specific names to the clusters for the getting started use-case, which is why I left it as is in the e2e script, but gave them generic names in the test because the clusters in the test are generic. However, not a strongly held opinion, if we want to make it completely generic we can go that route (or if we have better names to give them).